main.tf

```diff
@@ -1,18 +1,24 @@
-resource "helm_release" "aws-load-balancer-controller" {
+locals {
+  account_id = coalesce(var.account_id, data.aws_caller_identity.current.account_id)
+  region     = coalesce(var.region, data.aws_region.current.name)
+}
+
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "helm_release" "external_dns" {
   name       = "external-dns"
   namespace  = "kube-system"
   repository = "https://charts.bitnami.com/bitnami"
   chart      = "external-dns"
-  version    = "4.8.3"
-  # appVersion: 0.7.6
+  version    = var.helm_chart_version
```

Comment:
Hoeft voor mij persoonlijk niet, we gebruiken dit alleen hier.

Niet schadelijk maar niet nodig :)

Reply:
I see the value of this. It allows the user of the module to override the version, right?

Reply:
Ah in that case, we should just do this :)